tests/test_optimizely.py

```diff
@@ -1022,57 +1022,6 @@ def test_track__with_event_tags__forced_bucketing(self):
     self._validate_event_object(mock_dispatch_event.call_args[0][0], 'https://logx.optimizely.com/v1/events',
                                 expected_params, 'POST', {'Content-Type': 'application/json'})
 
-  def test_track__with_deprecated_event_value(self):
```

Comment:
Looks like your removal of this method decreased coverage for method isUserInExperiment. 
In other testcases we mock outer methods or isUserInExperiment so those lines do not get covered. Right a testcase to cover the missing lines.